data-es6.js

```diff
@@ -9375,23 +9375,53 @@ exports.tests = [
       },
     },
     {
-      name: 'JSON.stringify ignores symbols',
+      name: 'JSON.stringify ignores symbol primitives',
       exec: function() {/*
-        var object = {foo: Symbol()};
+        var object = { foo: Symbol() };
         object[Symbol()] = 1;
         var array = [Symbol()];
         return JSON.stringify(object) === '{}' && JSON.stringify(array) === '[null]' && JSON.stringify(Symbol()) === undefined;
       */},
       res: {
         babel: true,
         typescript: typescript.corejs,
-		es6shim: true,
+        es6shim: true,
         firefox36: true,
+        firefox40: true,
+        firefox44: true,
         chrome35: flag,
         chrome38: true,
         node012: true,
         webkit: true,
-      },
+      }
+    },
+    {
+      name: 'JSON.stringify ignores symbol objects',
+      exec: function() {/*
+        var testSymbolObject = function (sym) {
+          var object = { foo: sym };
+          try {
+            // some browsers throw a TypeError when setting symbol object keys.
+            // this isn't part of this test, so, ignore it if so.
+            object[sym] = 1;
+          } catch (e) {} // some browsers throw a TypeError when setting symbol object keys.
+          var array = [sym];
+          return JSON.stringify(object) === '{"foo":{}}' && JSON.stringify(array) === '[{}]' && JSON.stringify(sym) === '{}';
+        };
+        var objSym = Object(Symbol());
+        var symNoToJSON = Object(Symbol());
+        Object.defineProperty(symNoToJSON, 'toJSON', { enumerable: false, value: null }); // ensure it overrides the prototype, but is not callable
+        return testSymbolObject(objSym) && testSymbolObject(symNoToJSON);
+      */},
+      res: {
+        es6shim: true,
+        firefox40: true,
```

Comment:
What is the point of having multiple versions of the same browser here?  Isn't "firefox40:true" supposed to work for all versions >=40?